core/src/test/scala/unit/kafka/security/auth/ZkAuthorizationTest.scala

```diff
@@ -196,13 +196,12 @@ class ZkAuthorizationTest extends ZooKeeperTestHarness with Logging {
     // Additionally, we create the consumers znode (not in
     // securePersistentZkPaths) to make sure that we don't
     // add ACLs to it.
-    val secureOpt: String  = secondZk.isSecure match {
-      case true =>
-        firstZk.createPersistentPath(ZkUtils.ConsumersPath)
-        "secure"
-      case false =>
-        secondZk.createPersistentPath(ZkUtils.ConsumersPath)
-        "unsecure"
+    val secureOpt: String  = if (secondZk.isSecure) {
```

Comment:
The layout is a bit nicer if the if is moved to the following line. There's also two empty spaces before `=` instead of one (a previous issue).